File: android/app/src/main/java/com/bayesmech/camalytics/coverage/CoverageStats.kt
package com.bayesmech.camalytics.coverage

data class CoverageStats(
    val depthCoverage: Float = 0f,
    val accelerometerCoverage: Float = 0f,
    val gyroscopeCoverage: Float = 0f,
    val magnetometerCoverage: Float = 0f,
    val cameraIntrinsicsCount: Int = 0,
    val poseCoverage: Float = 0f,
    val inferredGeometryCoverage: Float = 0f,
    val gpsCoverage: Float = 0f,
    val averageFps: Float = 0f
)

File: ios/BayesMechVision/Coverage/CoverageStats.swift
import Foundation

struct CoverageStats: Sendable {
    var depthCoverage: Float = 0
    var accelerometerCoverage: Float = 0
    var gyroscopeCoverage: Float = 0
    var magnetometerCoverage: Float = 0
    var cameraIntrinsicsCount: Int = 0
    var poseCoverage: Float = 0
    var inferredGeometryCoverage: Float = 0
    var gpsCoverage: Float = 0
    var averageFPS: Float = 0
}
